--- gcc-9-2020-20200429-19AA7-src/gcc/ada/targparm.adb.orig     2021-04-12 15:38:05.372108235 +0300
+++ gcc-9-2020-20200429-19AA7-src/gcc/ada/targparm.adb  2021-04-12 15:38:15.584202824 +0300
@@ -334,14 +334,6 @@
             Opt.Locking_Policy          := 'C';
             goto Line_Loop_Continue;

-         --  Test for pragma Profile (Jorvik);
-
-         elsif Looking_At_Skip ("pragma Profile (Jorvik);") then
-            Set_Profile_Restrictions (Jorvik);
-            Opt.Task_Dispatching_Policy := 'F';
-            Opt.Locking_Policy          := 'C';
-            goto Line_Loop_Continue;
-
          --  Test for pragma Profile (GNAT_Extended_Ravenscar);

          elsif Looking_At_Skip
